.github/workflows/horus.yml

```diff
@@ -19,11 +19,12 @@ jobs:
     - name: Get the list of packages
       run: |
         cp jammy/packages_to_import /tmp/patched-packages
-    - name: Install Dependencies
+    - name: Install dependencies and enable sources
       run: |
         sed -i 's/^\(Types: deb\)$/\1 deb-src/g' /etc/apt/sources.list.d/ubuntu.sources
         apt update
-        apt install -y git python3-launchpadlib python3-apt python3-github
+        apt install -y git python3-launchpadlib python3-apt python3-github python3-pip python3-git
```

Comment:
Maybe we no longer need `python3-pip` because we don't install GitPython using it anymore?

Reply:
This is true.